From 5a67cd344529f2399fda191c9b2af978f71af3bc Mon Sep 17 00:00:00 2001
From: Wenping Zhang <wenping.zhang@rock-chips.com>
Date: Fri, 29 Sep 2017 09:16:54 +0800
Subject: [PATCH] ARM: dts: rockchip: amend fiq instead of irq debug for rk3229

This patch amends fiq instead of irq debug for rk3229 SoCs.

Change-Id: Ib910fb07c5836caf6c0ea73e7693cae2b2b7042d
Signed-off-by: Wenping Zhang <wenping.zhang@rock-chips.com>
---
 arch/arm/boot/dts/rk322x-android.dtsi | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/arch/arm/boot/dts/rk322x-android.dtsi b/arch/arm/boot/dts/rk322x-android.dtsi
index f8c69f275da4..b3d677993ab8 100644
--- a/arch/arm/boot/dts/rk322x-android.dtsi
+++ b/arch/arm/boot/dts/rk322x-android.dtsi
@@ -49,10 +49,11 @@
 
 	fiq-debugger {
 		compatible = "rockchip,fiq-debugger";
+		interrupts = <GIC_SPI 81 IRQ_TYPE_LEVEL_HIGH>;
 		rockchip,serial-id = <2>;
 		rockchip,signal-irq = <159>;
 		rockchip,wake-irq = <0>;
-		rockchip,irq-mode-enable = <1>; /* If enable uart uses irq instead of fiq */
+		rockchip,irq-mode-enable = <0>; /* If enable uart uses irq instead of fiq */
 		rockchip,baudrate = <1500000>; /* Only 115200 and 1500000 */
 		pinctrl-names = "default";
 		pinctrl-0 = <&uart21_xfer>;
@@ -161,10 +162,6 @@
 	status = "okay";
 };
 
-&uart2 {
-	status = "okay";
-};
-
 &usb_otg {
 	status = "okay";
 };
-- 
2.35.3

